R/utilities.R

```diff
@@ -166,13 +166,10 @@ runCommandline <- function(args, runid="01", make_bam=TRUE, del_sam=TRUE, dir=FA
           completed[[i]][[j]] <- file.exists(output(args)[[i]][[j]])
           names(completed[[i]][[j]]) <- output(args)[[i]][[j]]
           outputList <- c(outputList, output(args)[[i]][[j]])
+          names(outputList)[(names(outputList))==""] <- names(output(args)[i])
```

Comment:
Ha Ha Now I can't get isNA to work. It should be is.na

Reply:
Can you please explain better the issue and the fix, please?